gen/aircraft.py

```diff
@@ -1103,6 +1105,11 @@ def set_waypoint_tot(self, waypoint: MovingPoint, tot: int) -> None:
         waypoint.ETA = tot
         waypoint.ETA_locked = True
         waypoint.speed_locked = False
+    
+    def get_flight_point(self, name):
+        for i in self.flight.points:
+            if i.name == name:
```

Comment:
Instead of relying on the name of the waypoint (which seems like something we'd really easily break by accident), could you move this to `Flight` as `get_waypoint_by_type(self, waypoint_type: FlightWaypointType) -> Optional[FlightWaypoint]`?

Reply:
Flight.waypoint_with_type method already exists and performs same function, this isn't needed.

Reply:
Ha, even better. I knew I'd added that locally at _some_ point but thought maybe I hadn't merged it.